collect_app/src/main/java/org/odk/collect/android/tasks/GoogleSheetsAbstractUploader.java

```diff
@@ -227,7 +228,7 @@ private boolean uploadOneSubmission(String id, String instanceFilePath, String j
 
         // make sure column names are legal
         for (String n : columnNames) {
-            if (!isValidGoogleSheetsString(n)) {
+            if (!Regex.isValidGoogleSheetsString(n)) {
```

Comment:
Consider static imports for these methods, then the code will be the same as before, and perhaps more readable.